includes/class-wc-payments.php

```diff
@@ -467,12 +467,13 @@ public static function init() {
 		}
 
 		if ( WC_Payments_Features::is_express_checkout_enabled() ) {
-			add_action( 'woocommerce_after_add_to_cart_quantity', [ __CLASS__, 'display_express_checkout_button_separator_html' ], -1 );
-			add_action( 'woocommerce_proceed_to_checkout', [ __CLASS__, 'display_express_checkout_button_separator_html' ], -1 );
-			add_action( 'woocommerce_checkout_before_customer_details', [ __CLASS__, 'display_express_checkout_button_separator_html' ], -1 );
+			add_action( 'woocommerce_after_add_to_cart_quantity', [ __CLASS__, 'should_express_checkout_separator_be_displayed' ], -1 );
```

Comment:
Would it make sense if we change the method name to 

```suggestion
			add_action( 'woocommerce_after_add_to_cart_quantity', [ __CLASS__, 'display_express_checkout_separator_if_necessary' ], -1 );
```

or similar? Currently, the method name suggests that the return value is of type `boolean`, whereas it's not true.

Reply:
Yes, this method name is much better. Thanks for the suggestion!